crates/ecs/src/lib.rs

```diff
@@ -373,7 +374,8 @@ pub trait SystemParameter: Send + Sync + Sized {
     ) -> Option<Self>;
 
     /// A description of what data is accessed and how (read/write).
-    fn component_access() -> ComponentAccessDescriptor;
+    /// TODO: component_vec should not be locked for filters
```

Comment:
My suggestion is that we don't allow any `// todo` comments without them being tied to a specific issue, to ensure that we don't lose track of any to-dos. The format I've been using is `// todo(#nn): ...`.

Reply:
I think this will be fixed in this pull request when archetypes are ready. Should we allow `// todo` in drafts?

Reply:
Ah, I see. Yeah, that's fine!